src/index.html

```diff
@@ -11,7 +11,170 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+
+        <label name="surname">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          required
+          >
+        </label>
+
+        <label name="name">
+          Name:
+          <input
+          type="text"
+          name="name"
+          autocomplete="off"
+          required
+          >
+        </label>
+
+        <label name="age">
+          How old are You?
+          <input
+          type="number"
+          name="age"
+          required
+          min="1"
+          max="100"
+          value="12"
+          >
+        </label>
+
+        <label name="date-of-birth">
+          Full date of birth:
+          <input
+          type="date"
+          name="date-of-birth"
```

Comment:
[checklist p.4](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) use camelcase for values of name attribute